modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/ServerWorker.java

```diff
@@ -146,7 +147,7 @@ public void run() {
             ThreadContext.remove(CorrelationConstants.CORRELATION_MDC_PROPERTY);
             /* Subsequent to removing the correlation id MDC thread local value, a new value is put in case
                there is one */
-            if (sourceConfiguration.isCorrelationLoggingEnabled() && StringUtils.isNotEmpty(correlationId)) {
+            if ( PassThroughCorrelationConfigDataHolder.isEnable() && StringUtils.isNotEmpty(correlationId)) {
```

Comment:
is there a space after the parenthesis here?

Reply:
Will fix that